source errfunc1

function arbit(integer a, integer b, boolean t) to integer
begin
    leave 2
end

begin
    integer a;
    call arbit(1, 1, true)
end
